docs/README.md

```diff
@@ -93,6 +93,11 @@ var sdk = OpenTelemetrySdk.Create(builder => builder
     .WithMetrics(metrics => /* Metrics configuration goes here */)
     .WithTracing(tracing => /* Tracing configuration goes here */));
 
+// Optionally ForceFlush() telemetry objects in memory
```

Comment:
1. show shutdown here, no need to show flush.
2. Few lines above in this doc, it says Dispose() will gracefully shutdown and flush, so we need to add some wording here to indicate why an explicit shutdown() is needed and what scenarios requires using shutdown() as opposed to just relying on dispose().